string nextStep
int64 questionNum
int64 questionLevel
string questionType
string robotSpeech
string otherInfo
